inc/functions.php

```diff
@@ -562,51 +620,64 @@ function my_date($format, $stamp=0, $offset="", $ty=1, $adodb=false)
  * @param string $format The format of the email to be sent (text or html). text is default
  * @param string $message_text The text message of the email if being sent in html format, for email clients that don't support html
  * @param string $return_email The email address to return to. Defaults to admin return email address.
- * @return bool
+ * @return bool True if the mail is sent, false otherwise.
  */
 function my_mail($to, $subject, $message, $from="", $charset="", $headers="", $keep_alive=false, $format="text", $message_text="", $return_email="")
 {
-	global $mybb;
+	global $mybb, $plugins;
 	static $mail;
```

Comment:
Is there a need for this than `static $my_mailhandler;` above doesn't accomplish?

Reply:
Oh, that's right!